### ISSUE TITLE
Support for React 17 and React Native 0.68.2

### ISSUE THREAD

**Issue by diogotcorreia** [mak12/rn-faded-scrollview#8]
Currently, while using React Native 0.68.2, this package is not installable without using `--legacy-peer-deps`

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: rn-faded-scrollview@1.0.12
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   peer react@"*" from @gorhom/bottom-sheet@4.3.1
npm ERR!   node_modules/@gorhom/bottom-sheet
npm ERR!     @gorhom/bottom-sheet@"^4.3.1" from the root project
npm ERR!   peer react@"*" from @gorhom/portal@1.0.13
npm ERR!   node_modules/@gorhom/portal
npm ERR!     @gorhom/portal@"1.0.13" from @gorhom/bottom-sheet@4.3.1
npm ERR!     node_modules/@gorhom/bottom-sheet
npm ERR!       @gorhom/bottom-sheet@"^4.3.1" from the root project
npm ERR!   36 more (@jsamr/react-native-li, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"16.11.0" from rn-faded-scrollview@1.0.12
npm ERR! node_modules/rn-faded-scrollview
npm ERR!   rn-faded-scrollview@"^1.0.12" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: react@16.11.0
npm ERR! node_modules/react
npm ERR!   peer react@"16.11.0" from rn-faded-scrollview@1.0.12
npm ERR!   node_modules/rn-faded-scrollview
npm ERR!     rn-faded-scrollview@"^1.0.12" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Would it be possible to upgrade peer dependencies? Something like:
- React: `*`
- React Native: `>=0.62.2 <1.0.x`

Thanks

**Comment by mak12** [comment]
Hi @diogotcorreia 
Thank you for reporting this issue, will look into this on priority.

**Comment by mak12** [comment]
Hi @diogotcorreia ,
Thank you for the wait, its fixed in new release ([1.0.13](https://github.com/mak12/rn-faded-scrollview/releases/tag/1.0.13))
